src/ngx_http_lua_upstream_module.c

```diff
@@ -544,3 +548,43 @@ ngx_http_lua_upstream_find_upstream(lua_State *L, ngx_str_t *host)
 
     return NULL;
 }
+
+
+static int
+ngx_http_lua_upstream_current_upstream_name(lua_State *L)
+{
+    ngx_http_request_t *r;
+    ngx_http_upstream_t *us;
+    ngx_http_upstream_conf_t *ucf;
+    ngx_http_upstream_srv_conf_t *uscf;
```

Comment:
These variable names should be aligned up. Better use the `ngx-releng` tool to check your code style:

https://github.com/openresty/nginx-devel-utils